### ISSUE TITLE
Compatibilidade Delphi Seattle/Tokyo

### ISSUE THREAD

**Issue by ghost** [fabriciocolombo/delphi-rest-client-api#121]
Há compatibilidade com o Seattle/Tokyo ?

**Comment by fabioxgn** [comment]
@cgarrotti sim. Funciona normalmente. Se não me engano não dá para instalar os componentes pois agora o Delphi tem um componente com o mesmo nome. Mas se usar via código funciona tranquilo.

**Comment by ghost** [comment]
Eu fui tentar instalar, mas deu erro no DbxJsonUtils. Vou tentar usar via
codigo igual vc disse ai .
Valeu Fabio

Em 30 de agosto de 2018 10:27, Fábio Gomes <notifications@github.com>
escreveu:
